.github/workflows/ca-tests.yml

```diff
@@ -977,3 +977,292 @@ jobs:
+
+      - name: Install dependencies in primary container
+        run: docker exec primary dnf install -y 389-ds-base
+
+      - name: Install DS in primary container
+        run: docker exec primary ${PKIDIR}/ci/ds-create.sh
+
+      - name: Create DS signing cert in primary container
+        run: |
+          docker exec primary pki \
+              -d /etc/dirsrv/slapd-localhost \
+              -C /etc/dirsrv/slapd-localhost/pwdfile.txt \
+              nss-cert-request \
+              --subject "CN=DS Signing Certificate" \
+              --ext /usr/share/pki/server/certs/ca_signing.conf \
+              --csr ds_signing.csr
+          docker exec primary pki \
+              -d /etc/dirsrv/slapd-localhost \
+              -C /etc/dirsrv/slapd-localhost/pwdfile.txt \
+              nss-cert-issue \
+              --csr ds_signing.csr \
+              --ext /usr/share/pki/server/certs/ca_signing.conf \
+              --cert ds_signing.crt
+          docker exec primary pki \
+              -d /etc/dirsrv/slapd-localhost \
+              -C /etc/dirsrv/slapd-localhost/pwdfile.txt \
+              nss-cert-import \
+              --cert ds_signing.crt \
+              --trust CT,C,C \
+              Self-Signed-CA
+          docker exec primary certutil -L -d /etc/dirsrv/slapd-localhost
+
+      - name: Create DS server cert in primary container
+        run: |
+          docker exec primary pki \
+              -d /etc/dirsrv/slapd-localhost \
+              -C /etc/dirsrv/slapd-localhost/pwdfile.txt \
+              nss-cert-request \
+              --subject "CN=primary.example.com" \
+              --ext /usr/share/pki/server/certs/sslserver.conf \
+              --csr ds_server.csr
+          docker exec primary pki \
+              -d /etc/dirsrv/slapd-localhost \
+              -C /etc/dirsrv/slapd-localhost/pwdfile.txt \
+              nss-cert-issue \
+              --issuer Self-Signed-CA \
+              --csr ds_server.csr \
+              --ext /usr/share/pki/server/certs/sslserver.conf \
+              --cert ds_server.crt
+          docker exec primary pki \
+              -d /etc/dirsrv/slapd-localhost \
+              -C /etc/dirsrv/slapd-localhost/pwdfile.txt \
+              nss-cert-import \
+              --cert ds_server.crt \
+              Server-Cert
+          docker exec primary certutil -L -d /etc/dirsrv/slapd-localhost
+
+      - name: Enable DS security in primary container
+        run: |
+          docker exec primary dsconf localhost config replace nsslapd-security=on
+          docker exec primary dsctl localhost restart
+          docker exec primary sleep 10
+          docker exec -e LDAPTLS_CACERT=/ds_signing.crt primary ldapsearch -H ldaps://primary.example.com:636 -x -b "" -s base \* +
+
+      - name: Install CA in primary container
+        run: docker exec primary pkispawn -f /usr/share/pki/server/examples/installation/ca-secure-ds-primary.cfg -s CA -v
```

Comment:
nice to know that we have these examples.  Could you add info for this directory to the pkspawn man page and CS doc relevant section on installation?  It'd be very useful to readers I think.

Reply:
I added a link at the top of this page: https://github.com/dogtagpki/pki/wiki/Installation-Guide
Some specific installation pages already have direct links to the relevant files.

I think @PsOverflow is writing an installation page for this particular scenario. @PsOverflow, how is that going? We could add links to the deployment configuration files from that page.

I'd rather not add it into pkispawn man page since that page is already too long and quite outdated.
The man page is actually being converted into separate installation pages linked from the Installation Guide above.

Reply:
It's in progress @edewata, yeah I will point the links of the existing configuration in the same.